### DISCUSSION TITLE
Consider creating a Contributing Guide

### DISCUSSION THREAD

**Discussion by koksyn** [P3-model/P3-model#6] [Ideas]
This may not be a topic for today, but certainly for the coming months as the work progresses.
As this project is still in the making, there are a few areas to take care of at the beginning. One of these is the **Contributing Guide**.

It is important for OSS projects to involve more people in contribution. This guide should help them by explaining how contributors can submit pull requests, report issues, propose project improvements and enhancements, and where they can ask questions. The guide should clearly explain the project's workflow and standards for code, documentation, and other contributions, as clear set of rules for engagement and a step-by-step guide for creating satisfactory contributions. This guide often occurs in the form of a _CONTRIBUTING.md_ file in the root directory of a repository.

I have created this topic for future discussions, which will clarify the final form of this document (if we consider, of course, that there is already a need for it to be created).

**Comment by technites-pl** [comment]
Good idea. You can write first draft of such document if you have some ideas (and time of course) and then we can start discussion e.g. in merge request comments.
